sqlair.go

```diff
@@ -369,7 +491,31 @@ func (tx *TX) Query(ctx context.Context, s *Statement, inputArgs ...any) *Query
 	if ctx == nil {
 		ctx = context.Background()
 	}
+	if tx.isDone() {
+		return &Query{ctx: ctx, err: ErrTXDone}
+	}
 
 	qe, err := s.pe.Query(inputArgs...)
-	return &Query{qs: tx.tx, qe: qe, ctx: ctx, err: err}
+	if err != nil {
+		return &Query{ctx: ctx, err: err}
+	}
+
+	var ps *sql.Stmt
```

Comment:
A couple of nitpicks: a bit curious to have that long comment sandwished between the two variables that are used for the purpose of the code after the comment. Also, per the above this would be `sqlstmt`.

Reply:
Moved.